pkg/rest/request.go

```diff
@@ -71,14 +73,26 @@ func requestWithBody(method string, url string, body io.Reader, apiToken string)
 
 func executeRequest(request *http.Request) Response {
 	client := &http.Client{}
-	resp, err := client.Do(request)
-	if err != nil {
-		println("HTTP Request failed with Error: " + err.Error())
-		return Response{}
-	}
-	defer func() {
-		err = resp.Body.Close()
-	}()
-	body, err := ioutil.ReadAll(resp.Body)
-	return Response{resp.StatusCode, body}
+
+	rateLimitStrategy := createRateLimitStrategy()
```

Comment:
Would it maybe make sense to move this to a global var?

Reply:
You are right, makes more sense as a global var. "Global" in the package-global sense - it makes no sense to expose this outside of the `rest` package. Thanks!

Reply:
On second thought - we need to add a strategy for a simple exponential backoff (+jitter) soon. So, I'd rather leave it here as a preparation for later.